examples/dualstack/kubernetes-iptables.json

```diff
@@ -6,7 +6,6 @@
         },
         "orchestratorProfile": {
             "orchestratorType": "Kubernetes",
-            "orchestratorRelease": "1.20",
```

Comment:
@aramase is it important to include this 1.20 config here (for example, because this dualstack example doesn't work on versions older than 1.20) ?

Reply:
The dual-stack configurations should work for 1.17+. But we want to set to `1.20` because that's the recommended version for dual-stack.

Reply:
added back, thanks for clarifying!